docs/logic-apps/logicapp-authentication.md

```diff
@@ -0,0 +1,25 @@
+[home](../README.md)
+
+# Authenticating to Azure Logic App resource
+
+When using the Logic App testing features, you require a valid authentication setup.
```

Comment:
Let's put some structure in it instead of move as-is:
```suggestion
Before you can start using the Azure Logic App testing features, you need to authenticate with Microsoft Azure.

As of today, we provide the following authentication scenarios:

- [**Using Service Principal**](#using-a-service-principal)
```